apis/apps/v1beta4/status_types.go

```diff
@@ -36,6 +36,8 @@ type ConditionType string
 const (
 	ConditionRunning           ConditionType = "Running"
 	ConditionBlueGreenUpdating ConditionType = "BlueGreenUpdating"
+	ConditionProcess           ConditionType = "Process"
+	ConditionComplete          ConditionType = "Complete"
```

Comment:
Do not mix conditionTypes of different resources together.

Reply:
> Do not mix conditionTypes of different resources together.

done